identifier/backends/ldap/ldap.go

```diff
@@ -305,6 +305,7 @@ func NewLDAPIdentifierBackend(
 		if uri.Port() == "" {
 			addr += ":636"
 		}
```

Comment:
This change seems unrelated to the mr. At least it should be a dedicated commit.

Reply:
Done